k8s/apis/k8s/v1alpha1/vineyardd_webhook.go

```diff
@@ -74,11 +74,11 @@ func (r *Vineyardd) ValidateDelete() error {
 	return nil
 }
 
-// Make sure the image is valid
+// Make sure the vineyardd is valid
 func (r *Vineyardd) check() {
 	vineyarddlog.Info("validate image", "name", r.Name)
 	defaultImage := "docker.pkg.github.com/v6d-io/v6d/vineyardd:alpine-latest"
```

Comment:
Using the default image from dockerhub for publishing: `vineyardcloudnative/vineyardd:alpine-latest`.

Again, images on github packages should be used for CI only.